awx/main/dispatch/worker/callback.py

```diff
@@ -182,13 +184,15 @@ def flush(self, force=False):
             self.buff = {}
             self.last_flush = time.time()
             # only update metrics if we saved events
+            metrics_job_lag = 1  # TODO: currently a placeholder value because currently it is doing wacky stuff, a la maybe it's incrementing?
             if (metrics_bulk_events_saved + metrics_singular_events_saved) > 0:
```

Comment:
when we compute average probably safer to do it in this if block to avoid division by zero
```suggestion
            if (metrics_bulk_events_saved + metrics_singular_events_saved) > 0:
                metrics_avg_event_processing_seconds = metrics_total_job_event_processing_seconds / (metrics_bulk_events_saved + metrics_singular_events_saved)
```